acs-integration-tests/src/test/java/com/ge/predix/integration/test/PolicyEvaluationCachingIT.java

```diff
@@ -0,0 +1,519 @@
+            testPolicyName = this.policyHelper.setTestPolicy(this.acsAdminRestTemplate, this.acsZone1Headers, endpoint,
+                    policyFile);
+
+            ResponseEntity<PolicyEvaluationResult> postForEntity = this.acsAdminRestTemplate.postForEntity(
+                    endpoint + PolicyHelper.ACS_POLICY_EVAL_API_PATH,
+                    new HttpEntity<>(policyEvaluationRequest1, this.acsZone1Headers), PolicyEvaluationResult.class);
+            Assert.assertEquals(postForEntity.getStatusCode(), HttpStatus.OK);
+            PolicyEvaluationResult responseBody = postForEntity.getBody();
+            Assert.assertEquals(responseBody.getEffect(), Effect.PERMIT);
+
+            postForEntity = this.acsAdminRestTemplate.postForEntity(endpoint + PolicyHelper.ACS_POLICY_EVAL_API_PATH,
+                    new HttpEntity<>(policyEvaluationRequest2, this.acsZone1Headers), PolicyEvaluationResult.class);
+            Assert.assertEquals(postForEntity.getStatusCode(), HttpStatus.OK);
+            responseBody = postForEntity.getBody();
+            Assert.assertEquals(responseBody.getEffect(), Effect.PERMIT);
+
+            resource1.setAttributes(new HashSet<Attribute>(
+                    Arrays.asList(new Attribute[] { this.privilegeHelper.getAlternateOrgAttribute() })));
+            resource2.setAttributes(new HashSet<Attribute>(
+                    Arrays.asList(new Attribute[] { this.privilegeHelper.getAlternateOrgAttribute() })));
+            this.privilegeHelper.postResources(this.acsAdminRestTemplate, endpoint, this.acsZone1Headers, resource1,
+                    resource2);
+
+            postForEntity = this.acsAdminRestTemplate.postForEntity(endpoint + PolicyHelper.ACS_POLICY_EVAL_API_PATH,
+                    new HttpEntity<>(policyEvaluationRequest1, this.acsZone1Headers), PolicyEvaluationResult.class);
+            Assert.assertEquals(postForEntity.getStatusCode(), HttpStatus.OK);
+            responseBody = postForEntity.getBody();
+            Assert.assertEquals(responseBody.getEffect(), Effect.NOT_APPLICABLE);
+
+            postForEntity = this.acsAdminRestTemplate.postForEntity(endpoint + PolicyHelper.ACS_POLICY_EVAL_API_PATH,
+                    new HttpEntity<>(policyEvaluationRequest2, this.acsZone1Headers), PolicyEvaluationResult.class);
+            Assert.assertEquals(postForEntity.getStatusCode(), HttpStatus.OK);
+            responseBody = postForEntity.getBody();
+            Assert.assertEquals(responseBody.getEffect(), Effect.NOT_APPLICABLE);
+        } finally {
+            if (testPolicyName != null) {
+                this.policyHelper.deletePolicySet(this.acsAdminRestTemplate, this.zoneHelper.getZone1Url(),
+                        testPolicyName, this.acsZone1Headers);
+                this.privilegeHelper.deleteSubject(this.acsAdminRestTemplate, this.zoneHelper.getZone1Url(),
+                        subject.getSubjectIdentifier(), this.acsZone1Headers);
+            }
+        }
+    }
+
```

Comment:
(Minor) Typo. Rename testPolicyEvalCacheWhenResourcesChanges() to testPolicyEvalCacheWhenResourcesChange()